Vagrantfile

```diff
@@ -1,27 +1,35 @@
 Vagrant.configure("2") do |config|
-  config.vm.box = "ubuntu/xenial64"
+  config.vm.box = "centos/7"
 
   def create(config, hostname, ip)
     config.vm.define hostname do |host|
       host.vm.hostname = hostname
       host.vm.network "private_network", ip: ip
-      host.vm.provision "shell", inline: "echo ubuntu:ubuntu | chpasswd"
-      host.vm.provision "shell", inline: "apt-get update && apt-get install -y python-minimal"
+ #     host.vm.provision "shell", inline: "echo ubuntu:ubuntu | chpasswd"
```

Comment:
Probably better to just remove these instead of commenting them out.

Reply:
agree fixeddd

Reply:
done thanks